registry.json

```diff
@@ -4261,5 +4261,17 @@
     "telegram": "grahamde8313",
     "operatorName": "Derek Graham",
     "github": "grahamde8313"
+  },
+  {
+    "ticker": "ANTRX",
+    "poolId": "8b315cd9c1b128c5cfe70925a786f6db96cf220bbcea8a2a9c6afdb9",
+    "social": {
+      "website": "",
+      "twitter": "Antrix94841957",
```

Comment:
It would be nice to see some activity on this social media account. Right now there is one post and no one following or being followed.